caps/eth.md

```diff
@@ -52,6 +55,49 @@ a block near the head of the chain and downloads merkle tree nodes and contract
 incrementally by requesting the root node, its children, grandchildren, ... using
 [GetNodeData] until the entire tree is synchronized.
 
+### Block Propagation
+
+Newly-mined blocks must be relayed to all nodes. This happens through block announcements.
+
+Blocks are announced to all connected peers as soon as basic validity of the block has
```

Comment:
"announced to all connected"

This is false-ish. Blocks are propagated to sqrt(peers) after checking PoW. This ensures that the block ripples through the network if the PoW was valid. It however is not yet announced to everybody. After full block processing is finished and the block is saved to disk, only then is it announced. This is an important detail because the block **must** be in the database when announcing since we expect remote clients to immediately send requests for it that we need to be able to serve.